src/mapping/config/MappingConfiguration.cpp

```diff
@@ -81,8 +81,13 @@ MappingConfiguration::MappingConfiguration(
   }
   {
     XMLTag tag(*this, VALUE_RBF_GAUSSIAN, occ, TAG);
-    tag.setDocumentation("Local radial-basis-function mapping based on the Gaussian RBF with a cut-off threshold.");
-    tag.addAttribute(attrShapeParam);
+    tag.setDocumentation("Local radial-basis-function mapping based on the Gaussian RBF using a cut-off threshold.");
+    tag.addAttribute(makeXMLAttribute<double>(ATTR_SHAPE_PARAM, 0.)
+                         .setDocumentation("Specific shape parameter for RBF basis function."));
+    tag.addAttribute(makeXMLAttribute(ATTR_SUPPORT_RADIUS, 0.)
+                         .setDocumentation("Support radius of each RBF basis function (global choice)."));
+    tag.addAttribute(makeXMLAttribute<bool>(ATTR_USE_SHAPE_PARA, true)
+                         .setDocumentation("Select between the shape-parameter and the support-radius in order to determine the local cut-off of the Gaussian RBF."));
```

Comment:
The mapping would be easier to use if the configuration requires one of the two to be passed instead of having to specify which one to use.

We could set both to `NaN` by default and enforce that only one of the two is specified.
This should be easy to check.